public/app/features/browse-dashboards/components/RecentlyDeletedActions.tsx

```diff
@@ -104,32 +107,11 @@ export function RecentlyDeletedActions() {
     );
   };
 
-  const showDeleteModal = () => {
-    reportInteraction('grafana_delete_permanently_clicked', {
-      item_counts: {
-        dashboard: selectedDashboards.length,
-      },
-    });
-    appEvents.publish(
-      new ShowModalReactEvent({
-        component: PermanentlyDeleteModal,
-        props: {
-          selectedDashboards,
-          onConfirm: onDelete,
-          isLoading: isDeleteLoading,
-        },
-      })
-    );
-  };
-
   return (
     <Stack gap={1}>
       <Button onClick={showRestoreModal} variant="secondary">
         <Trans i18nKey="recently-deleted.buttons.restore">Restore</Trans>
       </Button>
-      <Button onClick={showDeleteModal} variant="destructive">
```

Comment:
Why are we removing option to perma delete?

Reply:
It's not possible to do that anymore, the dashboards will be deleted automatically after a certain time.